components/conjunctions/_screen-lg-min.scss

```diff
@@ -12,14 +12,14 @@
 		flex-direction: row;
 
 		&:not(.brdr-0) {
-			[class*="cnjnctn-col"] {
+			> [class*="cnjnctn-col"] {
```

Comment:
Since there is only one rule here, move sub rule to parent rule. Do this for line 10 also.